src/test/java/ch/liquidmind/inflection/compiler/util/InflectionCompilerTestUtility.java

```diff
@@ -0,0 +1,51 @@
+package ch.liquidmind.inflection.compiler.util;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.io.Files;
+
+import ch.liquidmind.inflection.compiler.CompilationJob;
+import ch.liquidmind.inflection.compiler.CompilationJob.CompilationMode;
+import ch.liquidmind.inflection.compiler.InflectionCompiler;
+import ch.liquidmind.inflection.loader.TaxonomyLoader;
+
+public final class InflectionCompilerTestUtility
+{
+	
+	/**
+	 * @param testClass defining the location of *.inflect files
+	 * @param inflectFilenames 
+	 */
+	public static CompilationJob createCompilationJob( Class<?> testClass, String... inflectFilenames ) throws URISyntaxException {
+		if (testClass == null) {
+			throw new IllegalArgumentException("testClass must not be null");
+		}
+		List< File > files = new ArrayList< >();
+		for (String resource : inflectFilenames) {
+			URL url = testClass.getResource( resource );
+			files.add( new File( url.toURI() ) );
```

Comment:
Remove the throws clause on this method ("throws URISyntaxException") and use the deflected API instead ("__URL.toURI( url )").